src/test/xenondb/README.md

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Just upload the files under the charts folder, the other files are useless.
You can reference other directory structure under src/test

Reply:
got it.